src/meson.build

```diff
@@ -303,6 +306,7 @@ if get_option('tests')
       '-DTESTDATADIR="' + testdatadir_src + ':' + testdatadir_dst + '"',
       '-DPLUGINBUILDDIR="' + pluginbuilddir + '"',
       '-DFU_OFFLINE_DESTDIR="/tmp/fwupd-self-test"',
+      '-DFU_MUTEX_DEBUG',
```

Comment:
This reminds me that some other aspects of the daemon are too verbose when called `--verbose`.  For example the `"using ------------- for $str"` and dispatching into plugins for usb and udev devices.  What do you think about putting those behind #defines as well?  Or having a concept of a daemon verbosity level?

Reply:
Well, how verbose is too verbose? It's too verbose when there's so much information you can't find the issue, but it's not verbose enough you can't pinpoint a failure. I don't think warning "levels" help much, but I do think we could separate this into various "classes" of debugging that can be enabled on end-user systems using either a cli option or env vars.

Reply:
Yeah classes of verbosity levels is really what I had in mind.  Perhaps making `--verbose` have an optional argument to remain compatible with previous behavior (no argument default to `all`)

Reply:
Some ideas in https://github.com/hughsie/fwupd/pull/738